packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -281,9 +281,15 @@ class TwinDeploymentHandler {
       let hru = 0;
       let sru = 0;
       let mru = 0;
-
+      let rootfs_size;
+      const disks: number[] = [];
       for (const workload of workloads) {
-        if (workload.type == WorkloadTypes.zmachine || workload.type == WorkloadTypes.zmount) {
+        if (workload.type == WorkloadTypes.zmachine) {
+          rootfs_size = workload.data["size"];
```

Comment:
you may have more than one vm per deployment, so you can't replace it. rootfs needs to be list as well